docs/fundamentals/code-analysis/style-rules/naming-rules.md

```diff
@@ -21,168 +21,109 @@ helpviewer_keywords:
 
-### Require a capitalization style
+> [!NOTE]
+>
+> If you're using a version of Visual Studio earlier than Visual Studio 2019 version 16.2, naming rules should be ordered from most-specific to least-specific in the EditorConfig file. The first rule encountered that can be applied is the only rule that is applied. However, if there are multiple rule *properties* with the same name, the most recently found property with that name takes precedence. For more information, see [File hierarchy and precedence](/visualstudio/ide/create-portable-custom-editor-options#file-hierarchy-and-precedence).
 
-To specify a particular capitalization style for symbol names, use this property:
+## Symbol group properties
 
-`dotnet_naming_style.<styleTitle>.capitalization = <value>`
+You can set the following properties for symbol groups, to limit which symbols are included in the group. To specify multiple values in a single property setting, separate them with a comma.
 
-The allowable values for this property are:
+| Property | Description | Allowed values | Required |
+| -- | -- | -- | -- |
+| `applicable_kinds` | Kinds of symbols in the group <sup>1</sup> | `*` (use this value to specify all symbols)<br/>`namespace`<br/>`class`<br/>`struct`<br/>`interface`<br/>`enum`<br/>`property`<br/>`method`<br/>`field`<br/>`event`<br/>`delegate`<br/>`parameter`<br/>`type_parameter`<br/>`local`<br/>`local_function` | Yes |
+| `applicable_accessibilities` | Accessibility levels of the symbols in the group | `*` (use this value to specify all accessibility levels)<br/>`public`<br/>`internal` or `friend`<br/>`private`<br/>`protected`<br/>`protected_internal` or `protected_friend`<br/>`private_protected`<br/>`local` (for symbols defined within a method) | Yes |
+| `required_modifiers` | Only match symbols with _all_ the specified modifiers <sup>2</sup> | `abstract` or `must_inherit`<br/>`async`<br/>`const`<br/>`readonly`<br/>`static` or `shared` <sup>3</sup> | No |
 
-- pascal_case
-- camel_case
-- first\_word_upper
-- all\_upper
-- all_lower
+**Notes:**
 
-> [!NOTE]
-> You must specify a capitalization style as part of your naming style, otherwise your naming style might be ignored.
+1. Tuple members aren't currently supported in `applicable_kinds`.
+2. The symbol group matches _all_ the modifiers in the `required_modifiers` property.  If you omit this property, no specific modifiers are required for a match. This means a symbol's modifiers have no effect on whether or not this rule is applied.
+3. If your group has `static` or `shared` in the `required_modifiers` property, the group will also include `const` symbols because they are implicitly `static`/`Shared`. However, you can create a new naming rule with its' own symbol group of `const` symbols.
```

Comment:
```suggestion
3. If your group has `static` or `shared` in the `required_modifiers` property, the group will also include `const` symbols because they are implicitly `static`/`Shared`. However, If you don't want the `static` naming rule to apply to `const` symbols, you can create a new naming rule with a symbol group of `const`.
```